lib/rpc.proto

```diff
@@ -1,6 +1,6 @@
 syntax = "proto3";
 
-//import "google/api/annotations.proto";
+import "google/api/annotations.proto";
```

Comment:
This needs to be commented out for the JavaScript grpc to work correctly.  Or at least it used to be.